src/fonts/load_petaluma.ts

```diff
@@ -4,7 +4,86 @@
 import { Font } from '../font';
 import { CommonMetrics } from './common_metrics';
 import { PetalumaFont } from './petaluma_glyphs';
+import { ChordSymbolGlyphMetrics } from '../chordsymbol';
+import { FontMetrics } from '../font';
 
+const petalumaChordMetrics: Record<string, ChordSymbolGlyphMetrics> = {
+  csymDiminished: {
+    leftSideBearing: -95,
+    advanceWidth: 506,
+    yOffset: 0,
+  },
+  csymHalfDiminished: {
```

Comment:
@rvilarl is this approach OK with you?

Before, we had a separate metrics file for the petaluma font.

With this PR, it is hard-coded into the load_petaluma.ts file.

I guess this can serve as a way to illustrate how we can customize metrics even though those files no longer exist.

Reply:
This is fine with me for vexflow4.